tests/kernel/mbox/mbox_usage/src/main.c

```diff
@@ -41,13 +42,9 @@ static void msg_sender(struct k_mbox *pmbox, k_timeout_t timeout)
 		mmsg.info = PUT_GET_NULL;
 		mmsg.size = 0;
 		mmsg.tx_data = NULL;
-		if (K_TIMEOUT_EQ(timeout, K_FOREVER)) {
-			k_mbox_put(pmbox, &mmsg, K_FOREVER);
-		} else if (K_TIMEOUT_EQ(timeout, K_NO_WAIT)) {
-			k_mbox_put(pmbox, &mmsg, K_NO_WAIT);
-		} else {
-			k_mbox_put(pmbox, &mmsg, timeout);
-		}
```

Comment:
Pfft.  Wonder how that happened, maybe this was a cocinelle or other automated thing that simplified this earlier to the redundant code?